### PR TITLE
Update 3D Secure page with latest developments

### PR DESCRIPTION
**Pull request by stevenjmesser**
- We know when 3DS1 is likely to be phased out
- We now support Worldpay's Exemption Engine
- Services are strongly encouraged to enable 3DS Flex (to get 3DS2 support) as soon as possible
- Most banks ask paying users to verify themselves with a code sent by SMS

### CODE REVIEW

File: source/3d-secure-2.html.erb [hunk 1]
```diff
@@ -83,7 +87,7 @@ title: GOV.UK Pay 3D Secure 2
         <li>corporate payments (unless the corporate card is in the name of an individual)</li>
       </ul>
 
-      <p class="govuk-body">Which exemptions are applied will depend on your Payment Service Provider (PSP).</p>
+      <p class="govuk-body">Which exemptions are applied will depend on your Payment Service Provider (PSP). See ‘How to set up 3D Secure 2’ for more details.</p>
```
Comment (oswaldquek) [inline source/3d-secure-2.html.erb:90]: Should ‘How to set up 3D Secure 2’ be linked?

Comment (stevenjmesser) [reply source/3d-secure-2.html.erb:90]: Could add an anchor link?

Comment (oswaldquek) [reply source/3d-secure-2.html.erb:90]: If there is one, yes

Comment (stevenjmesser) [reply source/3d-secure-2.html.erb]: OK, added with a new commit!

**Review by oswaldquek — APPROVED** [review]

**Pull request merged**